docker-compose.rocm.yml

```diff
@@ -0,0 +1,31 @@
+version: "3.9"
+services:
+  refact_self_hosted:
+    # TODO: figureout how to pass gpu to docker builds, so there is no need to install deepspeed at runtime
+    command: >
+      /bin/bash -c 'pip install deepspeed --no-cache-dir 
+      && python -m self_hosting_machinery.watchdog.docker_watchdog'
+    image: refact_self_hosting_rocm
+    build:
+      dockerfile: rocm.Dockerfile
```

Comment:
```
    build:
+      context: .
      dockerfile: rocm.Dockerfile
```
This was the only issue i found with this build so far :D I am testing it right now, just waiting for the models to download